requirements_all.txt

```diff
@@ -1017,7 +1017,7 @@ millheater==0.8.0
 minio==4.0.9
 
 # homeassistant.components.mitemp_bt
-mitemp_bt==0.0.3
+mitemp_bt==0.0.4
```

Comment:
Do we need to comment this in gen_requirements_all.py?

Reply:
I'm not sure if I'm following the reasoning behind your question?

Reply:
Pylint is failing.

Reply:
Oh lol, that I didn't notice yet :)

Reply:
I don't see what part of the requirements is causing that. bluepy and pygatt are optional in btlewrap.

Reply:
It's this line that has broken the package:
https://github.com/ratcashdev/mitemp/compare/v0.0.3...v0.0.4#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R22

Reply:
I'll submit a PR.

Reply:
https://github.com/ratcashdev/mitemp/pull/24

Reply:
https://pypi.org/project/mitemp-bt/0.0.5/ is released with the fix.

Reply:
wut? already! Almost didn't have time to blink on that one.
Thanks @MartinHjelmare 👍